parser/generator.go

```diff
@@ -162,6 +162,7 @@ func (g *Generator) GenerateDockerfile() error {
 
 	for _, phase := range phases {
 		if len(phase.commands) > 0 {
+			bufferRun.WriteString("source /.bzkenv\n")
```

Comment:
I changed the way to generate this in an upcoming PR 

The problem is still there but can be more easily fixed I guess

Any idea how to change this template to avoid the first iteration ? 
https://github.com/bazooka-ci/bazooka/blob/template_scripts/parser/template/bazooka_run.sh